src/components/BaseComponents/UserPopup/UserPopup.js

```diff
@@ -0,0 +1,44 @@
+import * as React from "react";
+import "../../../styles/components/popup.scss";
+
+const UserPopup = ({ popup, setPopup, user }) => {
+    console.log(user);
+    return (
+        <>
+            {popup ? (
+                <div className="popup">
+                    <div
+                        className="popup__close"
+                        onClick={() => setPopup(false)}
+                    >
+                        x
+                    </div>
+                    <div className="popup__container__left">
+                        <div className="popup__image">
+                            <img src={user.avatar_url} alt={user.login} />
+                            <div className="popup__login">{user.login}</div>
+                        </div>
+                        <div className="popup__location">{user.location}</div>
+                        <div className="popup__bio">{user.bio}</div>
+                        <div className="popup__number__repos">
+                            Public repositories: {user.public_repos}
+                        </div>
+                    </div>
+                    <div className="popup__container__right">
+                        <a
+                            href={user.html_url}
+                            target="_blank"
+                            rel="noreferrer"
+                            className="popup__link"
+                        >
+                            Go to Github
+                        </a>
+                    </div>
```

Comment:
Could we create two separate components here? Usually, the link should be reusable in many places. Try to think globally. It's not a big issue in that small project but take it into account that mostly you will need to work on an enormous project with thousand of components and tests. If you'll decide to chunk the project, the advantage of this solution will be invaluable.